landingpage/src/components/HeaderWithSearch/index.tsx

```diff
@@ -122,14 +130,11 @@ function HeaderWithSearch() {
 
 								<MyHr />
 
-								<PAExplanationView {...paExplanation} />
+								<TransactionTool title='Add or modify inscriptions' mode='inscribe' name={queryString}	history={history}/>
```

Comment:
Conceptually TransactionTool does not need to be wrapped by a DropDown:
`<DropDownContent title={Add or modify inscriptions}>
    <TransactionTool mode='inscribe' name={queryString}	history={history}/>
</DropDownContent>`